docs/application/native/guides/security/device-certificate-manager.md

```diff
@@ -0,0 +1,109 @@
+# Device Certificate Manager
+
+The Device Certificate Manager (DCM) provides cryptography services (digital certificates and keys) for authentication and secure communication with another system. Device certificate and key was embedded in device storage.
+
+The main features of the Device Certificate Manager API include:
+
+- Getting certificate chain
+
+- Creating digital signatures
+
+## Prerequisites
+
+To enable your application to use the Device Certificate Manager functionality:
+
+1. To use the Device Certificate Manager API (in [mobile](../../api/mobile/latest/group__CAPI__DEVICE__CERTIFICATE__MANAGER__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__DEVICE__CERTIFICATE__MANAGER__MODULE.html) applications), the application has to request permission by adding the following privilege to the `tizen-manifest.xml` file:
+
+   ```
+   <privileges>
+      <privilege>http://tizen.org/privilege/devicecertificate</privilege>
+   </privileges>
+   ```
+
+2. Before using the Device Certificate Manager API, check whether the device supports the device certificate feature with the `system_info_get_platform_bool()` function. If the device supports the Device Certificate Manager API, the function returns `true` in the second parameter.
+
+   ```
+   bool dcm_support;
+   system_info_get_platform_bool("http://tizen.org/feature/security.device_certificate", &dcm_support);
+   ```
+
+3. To use the functions and data types of the Device Certificate Manager API, include the `<device_certificate_manager.h>` header file in your application:
+
+   ```
+   #include <device_certificate_manager.h>
+   ```
+
+<a name="device_certificate_manager"></a>
+## Device Certificate Manager API usage example
```

Comment:
You had better divide two section. One is "Getting certificate chain", the other is "Creating digital signatures" as above main features description.

Reply:
Done.